<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardItemModel</class>
 <widget class="QMainWindow" name="StandardItemModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StandardItemModel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>tableView</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>plainTextEdit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionfile"/>
   <addaction name="actionsave"/>
   <addaction name="actionmodel_data"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_row"/>
   <addaction name="actionInsert_row"/>
   <addaction name="actionDelete_row"/>
   <addaction name="separator"/>
   <addaction name="actionLeft"/>
   <addaction name="actionCenter"/>
   <addaction name="actionRight"/>
   <addaction name="actionBold"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionfile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open3.bmp</normaloff>:/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save1.bmp</normaloff>:/icons/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionmodel_data">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/copy.bmp</normaloff>:/icons/copy.bmp</iconset>
   </property>
   <property name="text">
    <string>Model data</string>
   </property>
  </action>
  <action name="actionAdd_row">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/316.bmp</normaloff>:/icons/316.bmp</iconset>
   </property>
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="actionInsert_row">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/306.bmp</normaloff>:/icons/306.bmp</iconset>
   </property>
   <property name="text">
    <string>Insert row</string>
   </property>
  </action>
  <action name="actionDelete_row">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/delete1.bmp</normaloff>:/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/508.bmp</normaloff>:/icons/508.bmp</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/510.bmp</normaloff>:/icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/512.bmp</normaloff>:/icons/512.bmp</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/500.bmp</normaloff>:/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>StandardItemModel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
